aliases.sh

```diff
@@ -4,3 +4,4 @@ alias git-current-commit='git rev-parse --verify --short HEAD';
 alias git-current-tag='git fetch --tags --force --quiet && git describe --tags 2> /dev/null';
 alias git-reset='git reset --hard origin/$(git-current-branch)';
 alias git-clean-local-branches='git fetch --quiet && git branch -vv | grep -v  "^\*" | grep "\[.*: gone\]" | cut -d " " -f 3 | xargs git branch --delete';
+alias git-global-ignore='cp -i ~/utils/gitignore ~/.gitignore && git config --global core.excludesfile ~/.gitignore';
```

Comment:
Allow to personalize global ignore outside Utils but
- synchronisation is lost with ~/utils/gitignore changes
- replay of git-global-ignore can override customization if replying `y` to "overwrite ~/.gitignore? (y/n [n])"

Reply:
46450f2f5818375c6e79bb1cbc0eec8ecba45ea4
As soon as this global `core.excludesfile` is used for generic entries, it can be valuably committed.